packages/amplication-client/src/Resource/ResourceForm.tsx

```diff
@@ -150,7 +150,7 @@ function ResourceForm({ match }: Props) {
 
 export default ResourceForm;
 
-const UPDATE_RESOURCE = gql`
+export const UPDATE_RESOURCE = gql`
```

Comment:
why export ? where do you use it outside of resource form ?

Reply:
@barshimi  in ServiceConfigurationGitSettings component

Reply:
I moved the query to resources query file